lib/pytap13.py

```diff
@@ -102,8 +96,8 @@ def _parse(self, source):
                     in_yaml = True
                     continue
 
-            on_top_level = not line.startswith('    ')
-            raw_line = line.rstrip('\n')
+            on_top_level = not line.startswith(str_to_bytes('    '))
```

Comment:
Why not just `(b'....')`? Here and below.

Reply:
Maybe I didn't understand @Totktonada correctly. He said that there was a discussion about strings in the test-run and how to use them to support both python's versions. Consequently, you don't use literals associated with unicode and binary strings. So, I used str_to_bytes function. I was wrong?

Reply:
Fixed